docs/plugin-system.md

```diff
@@ -59,24 +59,16 @@ This plugin is registered to be rendered in the run header section (path /FLOW_I
 
 ## Plugin slots
 
-There are three implemented plugin slots. `run-header`, `task-details`, and `headless`. The desired slot must be given as a parameter to a plugin API register message.
+There are two implemented plugin slots, `run-header` and `task-details`. The desired slot must be given as parameter to a plugin API register message.
```

Comment:
it should probably mention plugin-api/Examples/hello-world/manifest.json and the definition of the slot for clarity